// @dopex-io/sdk/src/addresses/index.ts
const Addresses: { [key: number]: any } = {
  42: {
    WETH: '0x6701aa44353a8895b60b1289FeA326db66aCad9c',
    DPX: '0x6901a84FB66bE04E18c1b9181605FE4A7A027B01',
    RDPX: '0x9DDF1DC98A0690033EE03848Db9df49D6C52CA41',
    USDT: '0xf3e0d7bF58c5d455D31ef1c2d5375904dF525105',
    'DPX-WETH': '0xE665fe8014Af5734c52f925c9e2bee10b7225a55',
    'RDPX-WETH': '0x5E61F1ef81BCcE91b78E5b9cafDc00358A7C8C78',
    OptionPricing: '0x97d6dC71EE8Ee16adD0E908CFDEE14aFDa958ba5',
    TokenSale: '0x663C45241D9a85A5b48D2A75655759ee41b12847',
    StakingRewardsFactory: '0x915101231c8B79b2530B1aD95164dF6f5dEb9F76',
    DPXStakingRewards: '0x64Ceb676E09Be774D99Dd37329Ed0395e1c42c89',
    'DPX-WETHStakingRewards': '0xf765B799c19f92597f26899CB6dF0b1BEDc43b39',
    'RDPX-WETHStakingRewards': '0xBBA5c7C5A02D971fd9Ed3f52C89876affbe50CDa',
    SSOV: {
      Vault: '0x3e9c1CDCcFECb028436Ac06B36591E7E5c448c67',
      OptionPricing: '0xEA443E4516f02541ABC7644C8b1DFBc49F7189eC',
      IvOracle: '0x76CFd2F457fd6F210Fb44322255D35CBb67F42e7',
      PriceOracleAggregator: '0x48Bf3f10E0dB5E572ADaA3040Dd2f087619B3e99',
    },
  },
  1: {
    USDT: '0xdac17f958d2ee523a2206206994597c13d831ec7',
    USDC: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
    DAI: '0x6B175474E89094C44Da98b954EedeAC495271d0F',
    FRAX: '0x853d955aCEf822Db058eb8505911ED77F175b99e',
    MIM: '0x99D8a9C45b2ecA8864373A26D1459e3Dff1e17F3',
    GOHM: '0x0ab87046fBb341D058F17CBC4c1133F25a20a52f',
    DPX: '0xEec2bE5c91ae7f8a338e1e5f3b5DE49d07AfdC81',
    RDPX: '0x0ff5a8451a839f5f0bb3562689d9a44089738d11',
    WBTC: '0x2260fac5e5542a773aa44fbcfedf7c193bc2c599',
    WETH: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
    LINK: '0x514910771AF9Ca656af840dff83E8264EcF986CA',
    SPELL: '0x090185f2135308BaD17527004364eBcC2D37e5F6',
    CVX: '0x4e3fbd56cd56c3e72c1403e103b45db9da5b9d2b',
    CRV: '0xD533a949740bb3306d119CC777fa900bA034cd52',
    TokenSale: '0x468aC403f2C0aE0C65dB7dc22eD3440aC6ab0c39',
    'DPX-WETH': '0xf64af01a14c31164ff7381cf966df6f2b4cb349f',
    'RDPX-WETH': '0x0bf46ba06dc1d33c3bd80ff42497ebff13a88900',
    Tzwap1inchRouter: '0x02412065f205e5c4C72dD64F7b9b68e8749cf042',
    StakingRewardsFactory: '0x43a5cfB83d0DEcAaead90E0CC6DCA60A2405442B',
    DPXStakingRewards: '0xCE4D3E893f060Cb14B550B3E6B0AD512BEF30995',
    'DPX-WETHStakingRewards': '0x2a52330be21d311a7a3f40dacbfee8978541b74a',
    'RDPX-WETHStakingRewards': '0x175029c85B14C326C83c9f83D4A21Ca339F44Cb5',
    MerkleDistributor: '0x20E3D49241A9658C36Df595437160a6F6Dc01bDe',
  },
  421611: {
    USDT: '0x9A163588E2Db2a449655d5f04AAA7F0fC12dC3cb',
    USDTMock: '0xb26411F7515F94EbdB5152e396Fb0Ab8908423eA',
    Dopex: '0x4227933728ca1f40277CF078096B0fc0D91C2aDC',
    OptionPoolBroker: '0xaD170Fbe91f2CF533daF556C1cf5f610f27BdB79',
    WETH: '0xd530f42ec6a5Df88A82d51534FC6623E0721d6D3',
    WBTC: '0x1355A9AdD740eaFC3359A08484eFdb7Ad60D34BC',
    DPX: '0x9f2488ee66Fc770b6A98042fC67643bcA246df24',
    DopexRewards: '0x24f88c3c756dE421fe7C34938ee4a14EcBD9CEBC',
    OptionPoolFactory: '0xa142E07BC8E79bBf127bd6bd9cE07D4B8e5c6a54',
    OptionPoolRebates: '0xC36cB7188C9f3eF304f50cF9026f73CaA6E9d300',
    RDPX: '0x4A9E6EfCC15b2A80d51AD3f73623be9749d206a5',
    StakingVault: '0xcab883568E54568349a403d0bC1fFFeCaCc086e2',
    Delegator: '0xA2DC8638F282CB2c11b082E8dcbAFeBBa22af58D',
    WETHUSDTOptionPool: '0x8486c47f81723e65d453aC2B1d10417Ea408c95D',
    WBTCUSDTOptionPool: '0x5BA7168736FB4dfF1F35B207a5b1D4c87CDC7513',
    UniswapV2Factory: '0x929364CA2D16788dC891052f9eFff34b1D69c101',
    UniswapV2Router02: '0xdf8bF2fce78d14141fD66BD64a630b8C89f751E1',
    DopexOracle: '0xaac1d4bC61AC0685163694bf50A0A8bD491fA1e6',
    OptionPricingSvi: '0x31faa17Ed2E1639321f1aE5225F2dCDa237f133D',
    OptionPricingCustom: '0xC54d2511ac243e2552c64DA1A3DC78709393c762',
    VolumePoolFactory: '0x9E3bBa5568a78A1Ff3A5F07eb41f8638Cef86487',
    VolumePool: '0x4bc36d16CfE3c35ee1745779d6228d8B1c140491',
    LINK: '0x0d362e1ADCe13392367A5bBB6c3b831b6439b2Cc',
    YFI: '0x0A6D29d54196160590b72eD5c9525aaCAe5FCCce',
    FeeDistributor: '0xAfA515bCADE609daE9A0F31b6497d88118AA18c9',
    Router: '0xf8451E784f7dF8D18B259BF9088602CC02577A3b',
    AssetSwapper: '0xd2910146a6912A830E4a7369e5c1E93DE4225836',
    Margin: '0x94643579d1Bc148331708E3265b6536642e75416',
    StakingRewardsFactory: '0x36aAe3BDBb0b482af193dafF96A534106920B22e',
    DPXStakingRewards: '0x7cdA4869d9dc6A1da836064bFaB253559c4e91f7',
    Escrow: '0x2b4461c34e80058c3978cB5328367f3bEa23615B',
    SSOV: {
      Vault: '0x7Fe59162444B3B19246b78384861fE963C4F38E0',
      OptionPricing: '0xC88bE023Eb1c59589ACa5721fCd265e9cDc3142E',
      VolatilityOracle: '0x14D4d3049237D2c1E431055216DD3194Bd560684',
      PriceOracleAggregator: '0xc5555388Cd4f341dCC8c039e1EA63092f64b0B6f',
      Delegator: '0xd2dA8e59E715fC24df8D55CC4c855Ec8626422FE',
    },
    'SSOV-V3': {
      VIEWER: '0xDEe70dF134c7608B792d34cC1d93b00f320C3d1B',
      VAULTS: {
        'ETH-WEEKLY-CALLS-SSOV-V3-TEST':
          '0x29d1fdaDC3c24bdAB2f5944c762102b0Ae08a93c',
      },
    },
  },
  42161: {
    USDC: '0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8',
    DAI: '0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1',
    FRAX: '0x17FC002b466eEc40DaE837Fc4bE5c67993ddBd6F',
    MIM: '0xFEa7a6a0B346362BF88A9e4A88416B77a57D6c2A',
    DPX: '0x6c2c06790b3e3e3c38e12ee22f8183b37a13ee55',
    RDPX: '0x32eb7902d4134bf98a28b963d26de779af92a212',
    USDT: '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9',
    WETH: '0x82af49447d8a07e3bd95bd0d56f35241523fbab1',
    WBTC: '0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f',
    GOHM: '0x8D9bA570D6cb60C7e3e0F31343Efe75AB8E65FB1',
    JONES: '0x10393c20975cF177a3513071bC110f7962CD67da',
    GMX: '0xfc5A1A6EB076a2C7aD06eD22C90d7E710E35ad0a',
    '2CRV': '0x7f90122bf0700f9e7e1f688fe926940e8839f353',
    MAGIC: '0x539bde0d7dbd336b79148aa742883198bbf60342',
    'DPX-WETH': '0x0c1cf6883efa1b496b01f654e247b9b419873054',
    'RDPX-WETH': '0x7418f5a2621e13c05d1efbd71ec922070794b90a',
    StakingRewardsFactory: '0x923c88f0Aa5998953c01a23cDd1D82852f94Aeea',
    DPXStakingRewards: '0xc6D714170fE766691670f12c2b45C1f34405AAb6',
    RDPXStakingRewards: '0x125Cc7CCE81A809c825C945E5aA874E60ccCB6Bb',
    'DPX-WETHStakingRewards': '0x96B0d9c85415C69F4b2FAC6ee9e9CE37717335B4',
    'RDPX-WETHStakingRewards': '0x03ac1Aa1ff470cf376e6b7cD3A3389Ad6D922A74',
    AssetSwapper: '0xE5Ed878571c8592D61D1485471794d3EdB04e694',
    ERC20SSOV1inchRouter: '0xC296C505207E34FE8aFaC9d0B1ceD6ff17d0e89a',
    NativeSSOV1inchRouter: '0x444d91660642128A17dD574599981304f9C54DBD',
    '1inchRouter': '0x1111111254fb6c44bAC0beD2854e76F90643097d',
    DiamondPepesNFTMint: '0xcAD9297f00487a88Afa120Bf9F4823B52AE388b0',
    DiamondPepesNFT1inchRouter: '0x63D8De4079a8A46C721748E5db5f940a4C388Af5',
    DiamondPepesNFTPledge: '0xE974a44f859C1218fE0d8d151349788475692954',
    DiamondPepesNFTPledge2: '0x353e731EaA33fC1cc7f50E74EA390e95b192277F',
    Tzwap1inchRouter: '0xDA615781745fe52Fc3383Ad23cE9184419bEec30',
    Escrow: '0x6D9c288708a77d51Fa66393916511Cde60522baF',
    CurveGaugesOracle: '0xF45f20bc21C9933ae8613EdD8EF108b7fD25E527',
    NFTS: {
      DopexBridgoorNFT: '0x4Ee9fe9500E7C4Fe849AdD9b14beEc5eC5b7d955',
      DopexHalloweenNFT: '0x9baDE4013a7601aA1f3e9f1361a4ebE60D91B1B5',
      DiamondPepesNFT: '0xede855ceD3e5A59Aaa267aBdDdB0dB21CCFE5072',
    },
    'SSOV-V3': {
      VIEWER: '0x9abE93F7A70998f1836C2Ee0E21988Ca87072001',
      VAULTS: {
        'ETH-CALLS-SSOV-V3': '0x9E722E233646E1eDEa4A913489A75262A181C911',
        'ETH-WEEKLY-CALLS-SSOV-V3':
          '0x376bEcbc031dd53Ffc62192043dE43bf491988FD',
        'ETH-WEEKLY-CALLS-SSOV-V3-2':
          '0x2B055D25A5Fa6d493C53c178480E962Aabaa9255',
        'ETH-WEEKLY-CALLS-SSOV-V3-3':
          '0x3f0A22f98Df015FDD9B7eAAd29531956632d775a',
        'ETH-WEEKLY-CALLS-SSOV-V3-4':
          '0x9f52C2eeac80bb9084b4d67C26A56aAC7B3f54B0',
        'DPX-MONTHLY-CALLS-SSOV-V3':
          '0xDb62c01C23E247cea558E90814dBeB215b92C81F',
        'rDPX-MONTHLY-CALLS-SSOV-V3':
          '0x8D66b80E03fCeff5d825B77433B62400C00dB22D',
        'ETH-MONTHLY-CALLS-SSOV-V3':
          '0x7Af6399E4dd8C1D5B7C53dc2c6Ad04b39049c83E',
        'gOHM-MONTHLY-CALLS-SSOV-V3':
          '0xedA8F8D555a89d1ce5c631027ab89fD761D614F7',
        'ETH-WEEKLY-PUTS-SSOV-V3': '0xE1ACc3254bE1CcED13c8760A34E6D672021D0e02',
        'DPX-WEEKLY-PUTS-SSOV-V3': '0x459819C34266Dd9154172D373AC559Ce27f5F273',
        'rDPX-WEEKLY-PUTS-SSOV-V3':
          '0x4D7d6100122c92DDad3b57E472b00979e45a2326',
        'BTC-WEEKLY-PUTS-SSOV-V3': '0x411982342c0D41cEBB1706eD80ef32d48F3fEf3B',
        'gOHM-WEEKLY-PUTS-SSOV-V3':
          '0x7904eA09B0ce9A98Af8F4A2eD2f060364fB65042',
        'GMX-WEEKLY-PUTS-SSOV-V3': '0xB5Acc94D901eAB4206b49652bBf80E14ed8a79a0',
        'CRV-WEEKLY-PUTS-SSOV-V3': '0x7e00Ab584c3dcaB784FE8A04C471088C34386415',
        'LUNA-WEEKLY-PUTS-SSOV-V3':
          '0x29A1D81D7f3F3dC20E81Fe875012B7bED4011602',
        'DPX-MONTHLY-CALLS-SSOV-V3-2':
          '0x1ae38835Bf3afbEC178E8a59Ca82aA383dC3DF57',
        'rDPX-MONTHLY-CALLS-SSOV-V3-2':
          '0xAfD90Af84ae892C2DFeEA6D379087A7B4D21eC34',
        'ETH-MONTHLY-CALLS-SSOV-V3-2':
          '0x8161033b2776a9556b326D5eB468B2B76B7B4a23',
        'gOHM-MONTHLY-CALLS-SSOV-V3-2':
          '0x52c70138FfF1a5a6D821ab4Bb39D3dF0346f98bd',
        'ETH-WEEKLY-PUTS-SSOV-V3-2':
          '0xC7552Cd237823DeFa7F3a2E2cB6A3D0B9759F32C',
        'DPX-WEEKLY-PUTS-SSOV-V3-2':
          '0xE9132A503ba31Cf6320Ae97f42A9f3cf06Fa4e08',
        'rDPX-WEEKLY-PUTS-SSOV-V3-2':
          '0xdbAfE8d85620c2cfCc0b2a55d5d68D48B1d00631',
        'BTC-WEEKLY-PUTS-SSOV-V3-2':
          '0x54565213927794D7cA31436D01b799f487e204BA',
        'gOHM-WEEKLY-PUTS-SSOV-V3-2':
          '0x541fDD2284852Dacc5BA7E31241Ff5bc646b8985',
        'GMX-WEEKLY-PUTS-SSOV-V3-2':
          '0x7e513B848cFAF3Bc9FfB69a35981E5E1279acE59',
        'CRV-WEEKLY-PUTS-SSOV-V3-2':
          '0x03475494dc89d378C4268e90A62876eFb0278a1a',
        'DPX-MONTHLY-CALLS-SSOV-V3-3':
          '0x05E7ACeD3b7727f9129E6d302B488cd8a1e0C817',
        'rDPX-MONTHLY-CALLS-SSOV-V3-3':
          '0xd74c61ca8917Be73377D74A007E6f002c25Efb4e',
        'ETH-WEEKLY-PUTS-SSOV-V3-3':
          '0x32449DF9c617C59f576dfC461D03f261F617aD5a',
        'DPX-WEEKLY-PUTS-SSOV-V3-3':
          '0xf71b2B6fE3c1d94863e751d6B455f750E714163C',
        'rDPX-WEEKLY-PUTS-SSOV-V3-3':
          '0xb4ec6B4eC9e42A42B0b8cdD3D6df8867546Cf11d',
        'BTC-WEEKLY-PUTS-SSOV-V3-3':
          '0xa7507c48d78345475b85bc27B9CE9B84b354CaF7',
        'gOHM-WEEKLY-PUTS-SSOV-V3-3':
          '0x4269AF9076586230bF5fa3655144a5fe9CB877Fd',
        'GMX-WEEKLY-PUTS-SSOV-V3-3':
          '0xf071F0c56543A2671a2Dfc5FF51d5d858Be91514',
        'CRV-WEEKLY-PUTS-SSOV-V3-3':
          '0x7C5aC7E4E352B733CF65721d9Fe28A17Da890159',
        'DPX-WEEKLY-CALLS-SSOV-V3':
          '0x10FD85ec522C245a63239b9FC64434F58520bd1f',
        'rDPX-WEEKLY-CALLS-SSOV-V3':
          '0xCdaACF37726Bf1017821b5169e22EB34734B28A8',
        'gOHM-WEEKLY-CALLS-SSOV-V3':
          '0x546cd36F761f1D984eEE1Dbe67cC4F86E75cAF0C',
        'ETH-WEEKLY-CALLS-SSOV-V3-5':
          '0x9Cc9BeffE64868CB4B97890A19219449890E6eA0',
      },
    },
    SSOV: {
      DPX: {
        Vault: '0xbB741dC1A519995eac67Ec1f2bfEecbe5C02f46e',
        OptionPricing: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
        VolatilityOracle: '0xb6645813567bB5beEa8f62e793D075fE6d3Be0B1',
      },
      RDPX: {
        Vault: '0xA206a0531299821294A9BBa48726022D429c6892',
        OptionPricing: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
        VolatilityOracle: '0x3E0215c1D639280e13B46e3aF94Fb5630d1b3212',
      },
      ETH: {
        Vault: '0x2c9C1E9b4BDf6Bf9CB59C77e0e8C0892cE3A9d5f',
        OptionPricing: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
        VolatilityOracle: '0x87209686d0f085fD35B084410B99241Dbc03fb4f',
      },
      GOHM: {
        Vault: '0x54552CB564F4675bCEda644e47dE3E35D1c88E1b',
        OptionPricing: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
        VolatilityOracle: '0xbf91446115f3E3eaF5079A88E078F876C0d7A6A8',
      },
      GMX: {
        Vault: '0x5bE3c77ED3Cd42fc2c702C9fcd665f515862B0AE',
        OptionPricing: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
        VolatilityOracle: '0x83A5b587Ae36F342d405A7e5971941168E0adB5d',
      },
    },
    '2CRV-SSOV-P': {
      DPX: {
        Vault: '0xcD280E80DC3522F98eEF14ddcC00ebc30853b68c',
        OptionPricing: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
        VolatilityOracle: '0xAEAe470A71fAB319C88b38D21F6ADe73407dD3C0',
      },
      RDPX: {
        Vault: '0xC05200769EcB8ab6Cc3641397C6785f3A6D0a42A',
        OptionPricing: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
        VolatilityOracle: '0x3E0215c1D639280e13B46e3aF94Fb5630d1b3212',
      },
      ETH: {
        Vault: '0x134707f83168b9220c8d08447c1ba96a04797b96',
        OptionPricing: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
        VolatilityOracle: '0x36B25D2F1f638C384ac3E000beF154149d94638F',
      },
      GOHM: {
        Vault: '0xEd5dEdf81CCDC7c9F94Ef2330685dACF10656373',
        OptionPricing: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
        VolatilityOracle: '0xbf91446115f3E3eaF5079A88E078F876C0d7A6A8',
      },
      GMX: {
        Vault: '0x69617dBa76585230C14a2b94d3C42F4541b986a8',
        OptionPricing: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
        VolatilityOracle: '0x83A5b587Ae36F342d405A7e5971941168E0adB5d',
      },
      BTC: {
        Vault: '0xbD36e1F24D1bC7561cD82889F22058badB003792',
        OptionPricing: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
        VolatilityOracle: '0x90f5a93b562adba63b1C0F5358554e7272a28075',
      },
      CRV: {
        Vault: '0x9F2a0ffAA27a2F0FEf91Fd6DE19507a278C6302e',
        OptionPricing: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
        VolatilityOracle: '0xd89407796198799fA0EC640F7767B7441760386E',
      },
      LUNA: {
        Vault: '0x91D87F23c1Ca08e3d51bF2602F8E83514F34dA4C',
        OptionPricing: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
        VolatilityOracle: '0x92E0Da4bFd120EF25F74aF7041F48A205AD3bCDe',
      },
    },
    'RATE-VAULTS': {
      'MIM3CRV-1': '0x602B9f9B7126DEce2b9a589B49Fe5947c903A156',
      'MIM3CRV-2': '0x3BBCbe743AbeD14072EC26dABc4663Fa850f38D5',
      'PUSD3CRV-1': '0xdb2825f2A6c141A86862cCd5D4A86B18a436dd41',
      'PUSD3CRV-2': '0x7a839098c2754Dba5a8f489a73b8DB10BD3dc35c',
      'PUSD3CRV-3': '0x44BFe4979b2Ba2bfdac9CE224280129b90526048',
      'PUSD3CRV-BIWEEKLY-1': '0x66cF9033E119FcA8429bC87D423a14AaCEd22a3D',
      'PUSD3CRV-BIWEEKLY-2': '0xba3d2183A6378B80EC71D8CcFaFf33e44D32fa77',
    },
    DpxBonds: '0x8c44C0aB9A15bACAD7A4b663a89593c406C6B4EA',
    STRADDLES: {
      OPTION_PRICING: '0x2b99e3D67dAD973c1B9747Da742B7E26c8Bdd67B',
      Vault: {
        ETH: '0xE3E5ae751B35b985395d2d89C6475c75EeF67F19',
        RDPX: '0xCBCE47f554fBd4F8985800795322eF6901923e5c',
        DPX: '0x89801C1855D680a86aAF7Eb887Ff10d451683A8D',
      },
    },
  },
  56: {
    USDT: '0x55d398326f99059fF775485246999027B3197955',
    USDC: '0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d',
    DAI: '0x1dC56F2705Ff2983f31fb5964CC3E19749A7CBA7',
    FRAX: '0x90C97F71E18723b0Cf0dfa30ee176Ab653E89F40',
    MIM: '0xfE19F0B51438fd612f6FD59C1dbB3eA319f433Ba',
    WETH: '0x2170Ed0880ac9A755fd29B2688956BD959F933F8',
    WBNB: '0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c',
    WBTC: '0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c',
    VBNB: '0xA07c5b74C9B40447a954e1466938b865b6BBea36',
    ERC20SSOV1inchRouter: '0x3B0cfF305381c02465d93D48CcE122EBca668A23',
    '1inchRouter': '0x1111111254fb6c44bac0bed2854e76f90643097d',
    SSOV: {
      BNB: {
        Vault: '0x818ceD3D446292061913f1f74B2EAeE6341a76Ec',
        OptionPricing: '0x0ff5A8451A839f5F0BB3562689D9A44089738D11',
        VolatilityOracle: '0x32Eb7902D4134bf98A28b963D26de779AF92A212',
        PriceOracle: '0x0567F2323251f0Aab15c8dFb1967E4e8A7D42aeE',
        Router: '0x1Ae81A9ca1585c2Aca1D81A66AA5975Ba789BF96',
      },
    },
  },
  43114: {
    USDT: '0xc7198437980c041c805a1edcba50c1ce5db95118',
    USDC: '0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e',
    DAI: '0xd586e7f844cea2f87f50152665bcbc2c279d8d70',
    FRAX: '0xd24c2ad096400b6fbcd2ad8b24e7acbc21a1da64',
    MIM: '0x130966628846bfd36ff31a822705796e8cb8c18d',
    WETH: '0x49d5c2bdffac6ce2bfdb6640f4f80f226bc10bab',
    WBTC: '0x50b7545627a5162f82a992c33b87adc75187b218',
    NativeSSOV1inchRouter: '0xC198E8d7ad811F8C5Ed9b96210cDEc8f48e97C8c',
    '1inchRouter': '0x1111111254fb6c44bAC0beD2854e76F90643097d',
    SSOV: {
      AVAX: {
        Vault: '0x5540FEa353dF6302611DA1d57988104e43A4B6b6',
        OptionPricing: '0x20f9b64877eB0C1F180eb80321ee4f373842C714',
        VolatilityOracle: '0xEec2bE5c91ae7f8a338e1e5f3b5DE49d07AfdC81',
        PriceOracle: '0x0A77230d17318075983913bC2145DB16C7366156',
      },
    },
  },
  1088: {
    USDT: '0xbB06DCA3AE6887fAbF931640f67cab3e3a16F4dC',
    USDC: '0xEA32A96608495e54156Ae48931A7c20f0dcc1a21',
    DAI: '0x4651B38e7ec14BB3db731369BFE5B08F2466Bd0A',
    WETH: '0x420000000000000000000000000000000000000A',
    WBTC: '0xa5B55ab1dAF0F8e1EFc0eB1931a957fd89B918f4',
    METIS: '0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000',
    SSOV: {
      METIS: {
        Vault: '0x4Dcd650eea2E6ceF15691ceA6F693229d6226C6C',
        OptionPricing: '0xEec2bE5c91ae7f8a338e1e5f3b5DE49d07AfdC81',
        VolatilityOracle: '0x20f9b64877eB0C1F180eb80321ee4f373842C714',
        PriceOracle: '0x0ff5A8451A839f5F0BB3562689D9A44089738D11',
      },
    },
    'SSOV-V3': {
      VIEWER: '0xbf5EF951E62e9394B67349755372005D927Aa3A5',
      VAULTS: {
        'Metis-MONTHLY-CALLS-SSOV-V3':
          '0x9791c3A9A95b0a61e2FD43162E8077e2843dB16d',
      },
    },
  },
};

export default Addresses;
